vsdeinterlace/funcs.py

```diff
@@ -15,141 +15,97 @@
 )
 
 __all__ = [
-    'telop_resample',
+    'TelopResample',
     'FixInterlacedFades',
     'vinverse'
 ]
 
 
-class telop_resample(CustomIntEnum):
-    TXT60i_on_24telecined = 0
-    TXT60i_on_24duped = 1
-    TXT30p_on_24telecined = 2
+class TelopResample(CustomIntEnum):
+    IVTC_TXT60 = 0
+    DEC_TXT60 = 1
+    IVTC_TXT30 = 2
```

Comment:
Naming is hard, and I hope this will be documented, but FWIW I find the old names more understandable than the new ones. (Without docs, I can’t be certain that I actually understand them _correctly,_ though.)